examples/misc/lib/language_tour/functions.dart

```diff
@@ -26,6 +26,22 @@ void miscDeclAnalyzedButNotTested() {
     // #enddocregion function-shorthand
   }
 
+  {
+    // #docregion function-type
+    void printElement(int element) => print(element);
+
+    void doPrint(void Function(int) printFunction, {required List<int> list}) {
+      list.forEach(printFunction);
+    }
+
+    var list = [1, 2, 3];
+
+    // Store `doPrint` in a variable and call it.
+    void Function(void Function(int), {required List<int> list}) g = doPrint;
```

Comment:
Any particular reason for `g` as the variable name here?

Reply:
Do you have a better suggestion for the variable name?